opencti-platform/opencti-front/lang/en.json

```diff
@@ -1704,7 +1704,6 @@
   "INGESTION_MANAGER": "Ingestion manager",
   "PLAYBOOK_MANAGER": "Playbook manager",
   "FILE_INDEX_MANAGER": "File index manager",
-  "INDICATOR_DECAY_MANAGER": "Indicator decay manager",
```

Comment:
why has this translation been removed ?

Reply:
In must have executed an extract script in my branch, pre-rebase.
I'll check and restore the missing keys.

Reply:
I've done a review, all keys not related to confidence level have been restored (there was a couple)